cdc/puller/sorter/unified_sorter.go

```diff
@@ -198,6 +201,15 @@ func (s *UnifiedSorter) Run(ctx context.Context) error {
 
 		nextSorterID := 0
 		for {
+			// tentative value 1280000
+			for atomic.LoadInt64(&mergerBufLen) > 1280000 {
```

Comment:
When the merge buffer exceeds max length, could the deadlock also happen?

Reply:
No, it couldn't. Because blocking here does not block any other table from proceeding. @leoppro Can you confirm this?

Reply:
i think is fine, the blocking will not passing to other tables